index.html

```diff
@@ -21,7 +17,7 @@
             <div class="navbar">
                 <a href="#" class="navbar-text">My Logo</a>
                 <div class="menu-icon">
-                    <img src="./Image/Union.png" alt="">
+                    <img src="./Image/Union.png" alt="alt-image_text"> <!-- Added alternate image text-->
```

Comment:
- [ ] The purpose of the `alt` attribute is to describe the image for people who can't see the image. Kindly use a more descriptive alternative text, and please remove the comment after the `img` tag, since this does not add functionality to your code and after some time, it will not make sense to be present in this file.

Reply:
done.